aptos-move/framework/move-stdlib/sources/option.move

```diff
@@ -1,16 +1,14 @@
 /// This module defines the Option type and its methods to represent and handle an optional value.
 module std::option {
```

Comment:
can we add a comment about previous representation and that /v1/ rest endpoint is going to use that one to show them?